packages/expo-modules-core/ios/Swift/Logging/Logger.swift

```diff
@@ -35,6 +38,14 @@ public class Logger {
 
   // MARK: - Public logging functions
 
+  /**
+   Exposed for use by Swift wrappers
+    Application code should call one of the LogType-specific methods below
+   */
+  public func log(type: LogType = .trace, _ items: Any...) {
```

Comment:
I wanted to keep it separate and private since we provide specialized functions for each log type. It's just a helper. Do you really need it? I'd rather recommend to use other public functions.

Reply:
I'll have a look to see if there is a better way to do this

Reply:
After having a look at this, and making a few other changes, I'm going to close this PR and open a new one with my final set of changes (which will be smaller).